src/main/java/graphql/relay/Base64.java

```diff
@@ -1,24 +1,25 @@
 package graphql.relay;
 
 import java.nio.charset.Charset;
-import javax.xml.bind.DatatypeConverter;
-import java.io.UnsupportedEncodingException;
-
 
+/**
+ * @deprecated use {@link java.util.Base64} instead.
+ */
+@Deprecated
 public class Base64 {
```

Comment:
not in this package - but it is always good to be cautious before deleting a public class. having said that, it is fair enough to delete it in a breaking release.